layers/+emacs/tabs/funcs.el

```diff
@@ -0,0 +1,60 @@
+(defcustom spacemacs-tabs-auto-hide nil
+  "If non-nil hide tabs automatically after TABS-AUTO-HIDE-DELAY seconds."
+  :type '(boolean)
+  :group 'centaur-tabs)
+
+(defcustom spacemacs-tabs-auto-hide-delay 2
+  "Tabs auto hide delay in seconds."
+  :type '(float)
+  :group 'centaur-tabs)
+
```

Comment:
I like the approach with defcustom, I would even say this is preferable to the current approach with defvar. But please move it into the respective config.el file. Otherwise it will not be loaded at the right time.